consensus/src/runway/backup.rs

```diff
@@ -1,132 +1,153 @@
 use crate::{
+    alerts::Alert,
     units::{UncheckedSignedUnit, UnitCoord},
-    Data, Hasher, NodeIndex, Receiver, Round, Sender, SessionId, Signature, Terminator,
+    Data, Hasher, MultiKeychain, Multisigned, NodeIndex, Receiver, Round, Sender, SessionId,
+    Terminator,
 };
+
 use codec::{Decode, Encode, Error as CodecError};
 use futures::{channel::oneshot, FutureExt, StreamExt};
 use log::{debug, error, info, warn};
 use std::{
     collections::HashSet,
     fmt,
+    fmt::Debug,
     io::{Read, Write},
     marker::PhantomData,
 };
 
+const LOG_TARGET: &str = "AlephBFT-backup";
+
+#[derive(Clone, Debug, Decode, Encode, PartialEq)]
+pub enum BackupItem<H: Hasher, D: Data, MK: MultiKeychain> {
+    Unit(UncheckedSignedUnit<H, D, MK::Signature>),
+    AlertData(AlertData<H, D, MK>),
+}
+
+#[derive(Clone, Debug, Decode, Encode, PartialEq)]
+pub enum AlertData<H: Hasher, D: Data, MK: MultiKeychain> {
```

Comment:
This should probably be in `alerts` rather than here.

Reply:
Let me have it moved in the next pr